src/Annotation/SapiHandlerAnnotation.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\sapi\Annotation\SapiHandlerAnnotation.
+ */
+
+namespace Drupal\sapi\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Sapi handler annotation item annotation object.
+ *
+ * @see \Drupal\sapi\Plugin\SapiHandlerAnnotationManager
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class SapiHandlerAnnotation extends Plugin {
```

Comment:
I'd suggest to rename the class to `StatisticsPlugin` for now. The reason is that, while this class is a definition for annotation, it also defines how we define our plugins. IMHO `@StatisticsPlugin()` in plugin definition looks clearer than `@ SapiHandlerAnnotation()`.